src/call-api.ts

```diff
@@ -0,0 +1,152 @@
+import * as abi from "@api3/airnode-abi";
+import * as adapter from "@api3/airnode-adapter";
+import * as node from "@api3/airnode-node";
+import * as ois from "@api3/airnode-ois";
+import * as ethers from "ethers";
```

Comment:
Not sure if this makes a difference though but it seems a bit more consistent?
```suggestion
import { ethers } from "ethers";
```

Reply:
Wouldn't that look a bit weird in files like this one: https://github.com/api3dao/airkeeper/blob/bec128-api-calls/src/types.ts#L0-L1

I'm 💯 in favor of consistency but I'm not so sure how to achieve it with the imports tbh 😄

Reply:
Maybe I should just do:
```suggestion
import ethers from "ethers";
```